src/test-apps/PASEEngineTest.cpp

```diff
@@ -234,18 +244,14 @@ enum
 };
 
 //end PASEEngineTest Initialization
-void PASEEngineTest::Run()
+void PASEEngineTest::Run(void)
 {
+    PASEEngineTestContext *lContext;
     WEAVE_ERROR err;
-    WeavePASEEngine initiatorEng;
-    WeavePASEEngine responderEng;
```

Comment:
Why can't these be local?

Reply:
Because to ensure we don't run afoul of both Sapphire and RTOS -Wframe-size=X warnings, we have replicated this warning in Weave. These two plus the other two blow X by about 2x.

Since it's test code that isn't typically run on device (or at least devices that can't run multiple programs with main()), new/delete was an easy solve.

Reply:
I have it on my to-do list to remove the -Wframe-larger-than options from OpenWeave, as these can never be universally applied across all platforms.  (At a minimum it seems incorrect to limit a 64-bit pointer platform to using the same stack space as a 32-bit one).